app/views/includes/multi_results.scala.html

```diff
@@ -14,11 +14,12 @@
  * limitations under the License.
  *@
 
-@(calculationResponse:CalculationResponse, revalRateSubheader: Option[String], survivorSubheader: Option[String])(implicit messages: Messages)
+@import uk.gov.hmrc.govukfrontend.views.html.components.GovukTable
+@(calculationResponse:CalculationResponse, revalRateSubheader: Option[String], survivorSubheader: Option[String], govukTable : GovukTable)(implicit messages: Messages)
 
 @includes.result_header_subheader(calculationResponse, revalRateSubheader, survivorSubheader)
-
-@includes.results_table(calculationResponse)
+<br>
```

Comment:
Avoid using br tags - if additional spacing is need then there are override classes which can do this.

Reply:
Can you please share the syntax for that override class? I am not good at FE changes 🙈